### PR TITLE
Build doc deps

### PR DESCRIPTION
**Pull request by AnanyaKumar**

### CODE REVIEW

File: docs/build.md [hunk 1]
```diff
@@ -28,8 +34,14 @@ $GOPATH/src/github.com/google/cadvisor $ godep go test ./... -test.short
...
+## Running Built Binary
+
 Now you can run the built binary:
 
 ```
 $GOPATH/src/github.com/google/cadvisor $ sudo ./cadvisor
 ```
+
+If you plan to develop cAdvisor, read about the
+[Kubernetes Github Flow](https://docs.google.com/presentation/d/1HVxKSnvlc2WJJq8b9KCYtact5ZRrzDzkWgKEfm0QO_o/pub?start=false&loop=false&delayms=3000),
```
Comment (rjnagal) [inline docs/build.md]: I think we don't need this bit. The usual github help applies to cadvisor. Github bootcamp already covers this.

**Comment by rjnagal** [comment]
Thanks for improving the doc.

Just one minor nit. And you should also squash the two commits into one before we merge.

**Comment by AnanyaKumar** [comment]
@rjnagal Cool, done!

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**